contents/blog/0-to-hero.md

```diff
@@ -0,0 +1,186 @@
+  * anti goal - don't hire non technical people to scale support more cheaply
+
+Notice that the anti goals are all standard things that most companies follow, which is _exactly_ why bigger companies often become unpopular at scale.
+
+Haphazardly, we did experiment with one marketing channel – largely because I'm not good at coding, so figured I could help out this way. To do this, I first considered our users: engineers. I guessed they wouldn't appreciate being sold to, but that they would find following our journey interesting _and_ this was one topic I really could write about. So, I wrote about what my cofounder and I were up to – how we pivoted lots of times, or [what it was like going to San Francisco for YC](https://posthog.com/blog/moving-to-sf). This got us a couple more hits on the front page of Hacker News and thus we still do that to this day. So much so that to this day, now with over 60,000 companies using us, and I'm writing this blog post – as the CEO, at half midnight.
+
+
+## Getting to 1,000 users - do more of what is working, and clarify your ICP
+
+Some users now probably adore your product, others may be indifferent. The key is to figure what is different between the two types. It's [really important](https://newsletter.posthog.com/p/defining-our-icp-is-the-most-important) to [to find your Ideal Customer Profile](creating-ideal-customer-profile).
+
+In practice, we were at a larger user number before we focused on revenue _but_, if I were to repeat this process, I'd have done so sooner as we've changed [how we finance the business](https://posthog.com/handbook/people/finance), so we'd have always had better control of the business. You should do this _way_ earlier if you aren't totally certain you'll be able to make money with your idea.
+
+We started off thinking it'd look like this:
+
+* open source to give away value to the masses, which would generate lots of fans in companies big and small 
+* paid self-hosted version for big companies that didn't want to send data to a 3rd party
+
+This worked fairly well, we started growing quickly. However, at greater we should host it for people in cloud to make money (with a big free tier since cloud is a better experience for the majority of users, and good experience leads to more word of mouth growth).
+
+However, the cloud would have been very competitive for us. We didn't pick it in the first place as we had a lot of unicorn competitors.
+
+To stand out, however, we realized we could focus on building all the products in one (we started as product analytics, but we now have also got session replay, a/b testing, feature flags, web analytics, a data warehouse, a basic customer data platform and user surveys). The key was to not try to also go upmarket so people would have simpler needs. We could not replace Snowflake in a Fortune 500, but we could provide a warehouse for a 30 person startup, with simpler needs - for example.
```

Comment:
I feel like these paras (lines 135 to 144) could do with a big of a rejig. They're kind of all part of one thought, but it's a bit disordered atm – e.g. explaining why we didn't initially have a cloud product after saying we decided to launch one. It doesn't flow great atm.